### DISCUSSION TITLE
Collapsed Settings (How Do I Get Them To Stay Open?)

### DISCUSSION THREAD

**Discussion by gohan2091** [vladmandic/sdnext#1867] [Q&A] [answered]
With a recent update, things like CFG Scale, Clip Skip, Seed and Batch settings are now collpased and are found under tick boxes "Batch", "Seed Details" and "Advanced". How do I get these to stay open? I don't want to have to tick these everytime I open SD.Next. I have been through the settings and can't seem to find a way to revert back. Am I missing something? Please can someone show me specifically where these settings are changed?

**Comment by Sunsphire** [comment]
Settings > User interface defaults.

**Reply by gohan2091** [reply]
Then what do I do? I am still none the wiser.... The ui-config.json just reads:

`{
  "txt2img/Prompt/value": "cat",
  "txt2img/Sampling steps/value": 20,
  "img2img/Sampling steps/value": 20,
  "img2img/Scale/value": 1,
  "img2img/Mask blur/value": 4,
  "customscript/ultimate-upscale.py/img2img/Type/value": "Linear",
  "customscript/ultimate-upscale.py/img2img/Mask blur/value": 8,
  "customscript/ultimate-upscale.py/img2img/Padding/value": 32,
  "deforum_interface/Padding mode/value": "border"
}`

**Reply by Sunsphire** [reply, marked as answer]
That's a weirdly short ui-config.json to have but anyway, you don't have to actually change anything manually there. If you hover your mouse cursor over the "Set new defaults" you're going to see a really helpful hint on the top-right of the screen. You just make your user interface as you like it in your "From Text" tab for example, you tick the boxes that you want (for batch/seed details/advance/etc) and then you click "Set new defaults" in the 'settings>user interface defaults' and they become your new defaults. You can also press "View changes" in the 'settings > user interface defaults' to see what's being changed.

It also work with default sizes width and height, sampler, etc - for example I never use the 512x512 so i changed it to be 512x768 by default, I changed it to my most used sampler, secondary sampler, upscaler, etc

**Reply by gohan2091** [reply]
Amazing, thanks for explaining. That works well and my file is now a lot larger in content. My settings have saved and the settings don't collapse. :)

**Comment by AndrewRainfall** [comment]
Please put this info into FAQ category as well

**Reply by brknsoul** [reply]
Done!
https://github.com/vladmandic/automatic/discussions/1011#discussioncomment-9052955
